src/MTGCore/ViewModels/SearchCardViewModel.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace MTGCore.ViewModels
 {
-    public class FormViewModel
+    public class SearchCardViewModel
     {
-        public string Name { get; set; }
+        public SearchFilter SearchFilter { get; set; }
```

Comment:
I guess the benefit of this nested class over properties is to allow changes to the controller VM without affecting the internal SearchFilterMapper?